src/scenes/gui_dice_roller.cpp

```diff
@@ -0,0 +1,13 @@
+#include "gui_dice_roller.hpp"
+
+namespace meow {
+namespace {
+thread_local std::mt19937 rd_generator;
+}
```

Comment:
I thought you will define generator inside function (as it was mention in our earlier telegram discussion)